packages/create-plugin/templates/common/docker-compose.yaml

```diff
@@ -3,6 +3,7 @@ version: '3.0'
 services:
   grafana:
     container_name: '{{ pluginId }}'
+    platform: "linux/amd64"
```

Comment:
This change breaks prettier.

[It's set](https://github.com/grafana/plugin-tools/blob/main/.prettierrc) to single-quotes `platform: 'linux/amd64'`.

Maybe there should be a linter setup to run on PR's on this project? I also found another break at https://github.com/grafana/plugin-tools/pull/374#issuecomment-1884786130 ?

Reply:
Thanks for reporting this and it should be fixed in: https://github.com/grafana/plugin-tools/pull/657

Also, we have opened an issue to prevent malformed templates into the repo.